shuttle/templates/schedules/request_shuttle.html

```diff
@@ -9,33 +9,61 @@ <h4>Select a location and time below to request the shuttle:</h4>
+                <a class="dropdown-item" href="#">CLC</a>
+                <a class="dropdown-item" href="#">Anderson</a>
+                <a class="dropdown-item" href="#">Pine Tree</a>
+                <a class="dropdown-item" href="#">Kresge</a>
+                <a class="dropdown-item" href="#">Scandia</a>
+                <a class="dropdown-item" href="#">North</a>
+                <a class="dropdown-item" href="#">Rosedale Center</a>
+                <a class="dropdown-item" href="#">HealthPartner Clinic</a>
+                <a class="dropdown-item" href="#">Target</a>
+                <a class="dropdown-item" href="#">Lexington Station</a>
+                <a class="dropdown-item" href="#">Cub Foods</a>
             </div>
         </div>
-        <div class="align-side-by-side">
-            <div class="dropdown">
-                <button class="btn btn-primary btn-lg dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Time</button>
-                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                   <a class="dropdown-item" href="#">*Insert time here*</a>
-                    <a class="dropdown-item" href="#">*Insert more times here*</a>
-                </div>
-            </div>
-        </div>
-        <div class="align-side-by-side">
-            <div class="submit">
-                <input class="btn btn-warning btn-lg" type="submit" value="Request Shuttle">
-            </div>
+
+        <div class="submit">
+            <input class="btn btn-warning btn-lg submit-btn" type="submit" value="Request Shuttle">
         </div>
     </div>
 
     <div class="center-page">
-       <h4>Students awaiting shuttle: <span class="badge badge-light">*Insert number here*</span></h4>
+       <h4>Current Shuttle Waitlist: <span class="badge badge-pill badge-warning"></span></h4>
     </div>
+    <script>
+    function waitlist() {
+        // Updates waitlist
+        $.ajax({
+              url: "{{ url_for('SchedulesView:check_waitlist') }}",
+            }).done(function(data) {
+             $(".badge").text(data);
+            });
+    }
+    window.onload = waitlist();
+    $(document).ready(function() {
+        $(".dropdown-menu a").click(function(){
+            // Dropdown shows location selected
+            $(".location-menu:first-child").text($(this).text());
+            $(".location-menu:first-child").val($(this).text());
+        });
+        $('.submit-btn').click(function () {
+            // Sends selected location and user info to database. Updates waitlist
+            var data = {"location":$('.location-menu').val()}
+            $.ajax({
+                type: 'POST',
+                contentType: 'application/json',
+                url: "{{ url_for('SchedulesView:send_shuttle_request_path') }}",
+                data: JSON.stringify((data)),
+            }).done(function (data) {
```

Comment:
Add in the .fail() method to catch errors.

https://riptutorial.com/jquery/example/30084/jquery-ajax---success--error--vs---done-----fail--